build.gradle

```diff
@@ -6,6 +6,9 @@ apply plugin: 'io.codearte.nexus-staging'
 project.ext.preDexLibs = !project.hasProperty('disablePreDex')
 
 buildscript {
+    ext {
+        kotlin_version = '1.3.61'
+    }
```

Comment:
Remove this section. Use kotlinversion in line 12 also upgrade version to 1.3.72. Remember to update **kotlin_version** with **kotlinVersion**